lib/Service/CustomerService.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Stripe\Service;
+
+class CustomerService extends AbstractService
+{
+    const BALANCE_TRANSACTIONS = "balance_transactions";
+    const SOURCES = "sources";
+    const TAX_IDS = "tax_ids";
```

Comment:
I think we should prefix all those constants by something since we are also going to have various constants in all classes about types and such. Avoids a naming conflict with a purely internal constant. If we can we should even find a way to make those purely internal.

Reply:
I don't think we'll need other constants in the service classes. We might want to remove the constants entirely anyway once we codegen the service classes. Going to keep as-is for now, and we can revisit in another PR.

> If we can we should even find a way to make those purely internal.

Support for private constants was added in PHP 7.1 :p